python/src/rtree.rs

```diff
@@ -287,12 +291,11 @@ impl RTree {
     fn from_interleaved(
         _cls: &Bound<PyType>,
         py: Python,
-        boxes: PyObject,
+        boxes: pyo3_arrow::PyArray,
```

Comment:
This can be import shortened now to `PyArray` because we only have one in our crate now.